controllers/healthcheck_controller.go

```diff
@@ -347,6 +501,59 @@ func (r *HealthCheckReconciler) watchWorkflowReschedule(ctx context.Context, log
 	return nil
 }
 
+func (r *HealthCheckReconciler) watchRemedyWorkflow(log logr.Logger, wfNamespace, wfName string, hc *activemonitorv1alpha1.HealthCheck) error {
+	var now metav1.Time
+	then := metav1.Time{Time: time.Now()}
+	//repeatAfterSec := hc.Spec.RepeatAfterSec
```

Comment:
This should not be commented out, right?

Reply:
we donot need a repeatAfterSec for remedy as the expectation is to run remedy once..
if we need to run more than once we can use retrystrategy in the workflow.